tool/tctl/common/edit_command_test.go

```diff
@@ -42,7 +42,8 @@ func TestEditResources(t *testing.T) {
 	t.Parallel()
 	log := utils.NewLoggerForTests()
 	fc, fds := testhelpers.DefaultConfig(t)
-	_ = testhelpers.MakeAndRunTestAuthServer(t, log, fc, fds)
+	authServer := testhelpers.MakeAndRunTestAuthServer(t, log, fc, fds)
+	t.Cleanup(func() { require.NoError(t, authServer.Close()) })
```

Comment:
Can we register the cleanup inside the function that returns the auth server? It already takes t.